core.go

```diff
@@ -1,23 +1,47 @@
 package zapsentry
 
 import (
+	"errors"
 	"strings"
 	"time"
 
 	"github.com/getsentry/sentry-go"
+	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
+const (
+	maxLimit = 1000
+
+	zapSentryScopeKey = "_zapsentry_scope_"
+)
+
+func NewScope() zapcore.Field {
+	f := zap.Skip()
+	f.Interface = sentry.NewScope()
```

Comment:
I think we should allow users to pass their own scope, as it may be used to add additional context data.

```
newLogger.With(zapsentry.Scope(myScope))
// or
newLogger.With(zapsentry.Scope(sentry.NewScope()))
```